package.json

```diff
@@ -76,7 +76,7 @@
     "@metamask/ethjs-contract": "^0.3.3",
     "@metamask/ethjs-filter": "^0.2.0",
     "@metamask/ethjs-provider-http": "^0.2.0",
-    "@metamask/ethjs-query": "^0.5.1",
```

Comment:
I don't see any of these versions in the CHANGELOG for `@metamask/ethjs-query`. Were these backport releases, and what is the significance of this bump?